src/index.js

```diff
@@ -13,9 +13,9 @@ export default options => {
 
   /**
    * Get rooms socket is subscribed to.
-   *
-   * @param {String} id
-   * @param {Function} [fn]
+   * Used for get rooms
```

Comment:
But why not just document everything clearly, so when using IDEs it picks up the documentation and make the development process much easier?